### PR TITLE
Airtable form

### PR DESCRIPTION
**Pull request by backflip**
This PR adds the Airtable integration necessary for our two Call for Speakers forms. Styles need to be optimized, but it is functional.

- Regular: https://frontconference-website-git-feat-437759-front-conference-zurich.vercel.app/call-for-speakers-submission
- Bootcamp: https://frontconference-website-git-feat-437759-front-conference-zurich.vercel.app/call-for-speakers-bootcamp-application

It additionally uses environment variables (already saved to Vercel) for Contentful.

### CODE REVIEW

**Comment by uncenteredDiv** [comment]
@backflip it seems like your IDE is not taking the `tab` config into account - since you are reformatting quite a few lines ... and it seems like they are massively indented.

**Comment by backflip** [comment]
> @backflip it seems like your IDE is not taking the `tab` config into account - since you are reformatting quite a few lines ... and it seems like they are massively indented.

Not sure I understand: The `.editorconfig` specifies `indent_style = tab` and files like `Button.astro` seem to use spaces. What is the expected formatting behavior?

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-lts/*
+20
```
Comment (backflip) [inline .nvmrc:1]: This makes sure the version corresponds to the one Vercel provides (as they are always late with LTS versions).

File: README.md [hunk 2]
```diff
@@ -30,12 +30,16 @@ Any static assets, like images, can be placed in the `public/` directory.
...
-| `pnpm install`          | Installs dependencies                              |
-| `pnpm run dev`          | Starts local dev server at `localhost:3000`        |
-| `pnpm run build`        | Build your production site to `./dist/`            |
-| `pnpm run preview`      | Preview your build locally, before deploying       |
-| `pnpm run start`        | Starts a production dev server at `localhost:3000` |
-| `pnpm run astro ...`    | Run CLI commands like `astro add`, `astro preview` |
-| `pnpm run astro --help` | Get help using the Astro CLI                       |
+| Command                | Action                                             |
+| :--------------------- | :------------------------------------------------- |
+| `npm install`          | Installs dependencies                              |
```
Comment (backflip) [inline README.md:35]: Detail: I could not find any `pnpm` setup, so I'm assuming everyone is using npm.

**Comment by uncenteredDiv** [comment]
> > @backflip it seems like your IDE is not taking the `tab` config into account - since you are reformatting quite a few lines ... and it seems like they are massively indented.
> 
> Not sure I understand: The `.editorconfig` specifies `indent_style = tab` and files like `Button.astro` seem to use spaces. What is the expected formatting behavior?

hmm ... seems like it's only Github's Display Style 🤷‍♂️ 
It thought you have like a massive indention 🙈

**Review by uncenteredDiv — APPROVED** [review]

**Pull request merged**